dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -160,28 +160,43 @@ def test_true_categorical_report(self):
         self.assertTrue(report["categorical"])
         six.assertCountEqual(
             self,
-            ['unique_count', 'unique_ratio', 'categories'], report['statistics']
+            ['unique_count', 'unique_ratio', 'categories', 'categorical_count'], report['statistics']
         )
         self.assertEqual(3, report["statistics"]["unique_count"])
         self.assertEqual(0.25, report["statistics"]["unique_ratio"])
         self.assertCountEqual(
             ["a", "b", "c"], report["statistics"]["categories"]
         )
+        expected_dict = {"a": 3, "b": 4, "c": 5}
+        self.assertTrue(report["statistics"]["categories"])
+        self.assertEqual(expected_dict,
+                             report["statistics"]["categorical_count"])
 
     def test_false_categorical_report(self):
         df_non_categorical = pd.Series(list(map(str, range(0, 20))))
         profile = CategoricalColumn(df_non_categorical.name)
         profile.update(df_non_categorical)
 
         report = profile.profile
+        report.pop("times")
```

Comment:
lets still add a check to make sure not None, e.g. `self.assertIsNotNone(report.pop("times, None))`

Reply:
will validate something at least exists and that the key exists

Reply:
same as above